### DISCUSSION TITLE
Questions about the xamarin.android binding.

### DISCUSSION THREAD

**Discussion by mcguu** [xamarin/Xamarin.Forms#14107] [Q&A]
I am trying to binding `.aar` to the xamarin.android. and there is some errors there are one I can not understand and resolve.
in the `.aar` there is an interface:

**Comment by EmilAlipiev** [comment]
ask your question here and tag Xamarin.Forms and Xamarin https://docs.microsoft.com/en-us/answers/products/dotnet?WT.mc_id=xamarinforumsredirect-web-xamarin
